packages/api-gateway/src/drivers/http/routes/solicitudes/index.js

```diff
@@ -0,0 +1,39 @@
+const { solicitudesAdapter } = require('../../adapters');
```

Comment:
I think it could be better in the adapters folder, divide each adapters by entities. But in this case, the router file is related to the root directory to build the path of the endpoint.
I made some changes but I think this directory has to be unique